src/figuresClasses.ts

```diff
@@ -1,19 +1,79 @@
 export interface Figure {
-
+  color: string,
+  getArea(): number,
+  a?: number,
+  b?: number,
+  c?: number,
+  radius?: number,
+  width?: number,
+  height?:number,
+  shape: string,
 }
 
-export class Triangle {
+export class Triangle implements Figure {
+  shape: string = 'triangle';
 
-}
+  constructor(
+    public color: string,
+    public a: number,
+    public b: number,
+    public c: number,
+  ) {
+    this.getArea();
+  }
 
-export class Circle {
+  getArea():number {
+    if (this.a + this.b > this.c) {
+      if (this.b + this.c > this.a) {
+        if (this.a + this.c > this.b) {
+          const perimetr = (this.a + this.b + this.c) / 2;
 
+          return Math.floor(Math.sqrt(perimetr * (perimetr - this.a)
+          * (perimetr - this.b) * (perimetr - this.c)) * 100) / 100;
+        }
+      }
+    }
+
```

Comment:
Remove this line, as the error will be thrown in the constructor after suggested changes.